tests/integration/update_cluster/minimal_scaleway/data/scaleway_instance_server_control-plane-fr-par-1-0_user_data

```diff
@@ -130,7 +130,7 @@ ClusterName: scw-minimal.k8s.local
 ConfigBase: memfs://tests/scw-minimal.k8s.local
 InstanceGroupName: control-plane-fr-par-1
 InstanceGroupRole: ControlPlane
-NodeupConfigHash: Eju0QG9LIx4osURNz5K7RCCbZ5hWBWrmyJ83jQfXFeE=
+NodeupConfigHash: qkMcgEhTy3oHXOw6o8xCcTcSY7T7PLlQIesc5/8Ntvg=
```

Comment:
(Aside): I'm starting to wonder if we should just regexp replace these values in the golden files :-)